test/built-ins/Math/hypot/Math.hypot_ToNumberErr.js

```diff
@@ -13,26 +13,23 @@ info: |
   3. For each element _number_ of _coerced_, do
 ---*/
 
-let uniqueErrorCount = 0;
-class UniqueError extends Error {
-  constructor() { super(); uniqueErrorCount++; }
-}
+var counter = 0;
 
 assert.throws(
-  UniqueError,
+  Test262Error,
   function() {
     Math.hypot(
       Infinity,
       -Infinity,
       NaN,
       0,
       -0,
-      {valueOf(){ throw new UniqueError(); }},
-      {valueOf(){ throw new UniqueError(); }},
+      {valueOf: function(){ throw new Test262Error(); }},
+      {valueOf: function(){ counter++; }}
```

Comment:
In the future: changing to _MethodDefinition_ syntax to _PropertyName_ `:` _AssignmentExpression_ syntax is unnecessary, please don't, unless you're specifically testing the latter syntax.

Reply:
We're not testing either syntax. The change is necessary for the purpose of simplification--the stated goal of this patch.